src/utils/message.ts

```diff
@@ -0,0 +1,80 @@
+export const MessageType: {[type: string]: number} = {
+    UI_GET_DATA: 0,
+    UI_GET_ACTIVE_TAB_INFO: 1,
+    UI_SUBSCRIBE_TO_CHANGES: 2,
+    UI_UNSUBSCRIBE_FROM_CHANGES: 3,
+    UI_CHANGE_SETTINGS: 4,
+    UI_SET_THEME: 5,
+    UI_SET_SHORTCUT: 6,
+    UI_TOGGLE_URL: 7,
+    UI_MARK_NEWS_AS_READ: 8,
+    UI_LOAD_CONFIG: 9,
+    UI_APPLY_DEV_DYNAMIC_THEME_FIXES: 10,
+    UI_RESET_DEV_DYNAMIC_THEME_FIXES: 11,
+    UI_APPLY_DEV_INVERSION_FIXES: 12,
+    UI_RESET_DEV_INVERSION_FIXES: 13,
+    UI_APPLY_DEV_STATIC_THEMES: 14,
+    UI_RESET_DEV_STATIC_THEMES: 15,
+
+    BG_CHANGES: 16,
+    BG_ADD_CSS_FILTER: 17,
+    BG_ADD_STATIC_THEME: 18,
+    BG_ADD_SVG_FILTER: 19,
+    BG_ADD_DYNAMIC_THEME: 20,
+    BG_EXPORT_CSS: 21,
+    BG_UNSUPPORTED_SENDER: 22,
+    BG_CLEAN_UP: 23,
+    BG_RELOAD: 24,
+    BG_FETCH_RESPONSE: 25,
+
+    CS_COLOR_SCHEME_CHANGE: 26,
+    CS_FRAME_CONNECT: 27,
+    CS_FRAME_FORGET: 28,
+    CS_FRAME_FREEZE: 29,
+    CS_FRAME_RESUME: 30,
+    CS_EXPORT_CSS_RESPONSE: 31,
+    UI_SAVE_FILE: 32,
+    UI_REQUEST_EXPORT_CSS: 33,
+    CS_FETCH: 34,
+};
+
+// This map exists only for debugging convenience and is not used in the final build output.
+export const MessageTypeDescriptions = {
+    [MessageType.UI_GET_DATA]: 'ui-get-data',
```

Comment:
Why don't you use strings, but numbers in the "enum"? Is that a micro optimisation? Come on :)